documentation/book/con-user-operator-cluster-label.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// assembly-getting-started-user-operator.adoc
+
+[id='con-user-operator-cluster-label-{context}']
+
+= Identifying a Kafka cluster for user creation
```

Comment:
All the comments about handling and not just creation are applicable here.

Reply:
Changed to: 'Identifying a Kafka cluster for user handling'